env/ups.go

```diff
@@ -1,49 +1,40 @@
 package env
 
 import (
-	"log"
+	"fmt"
 
 	"github.com/cloudfoundry-community/go-cfenv"
 )
 
 // WithUPSLookup configures the VarSet to use the Cloud Foundry user-provided
 // service with the given name as a lookup source.
-func WithUPSLookup(logger *log.Logger, app *cfenv.App, name string) VarSetOpt {
+func WithUPSLookup(app cfenv.App, name string) VarSetOpt {
```

Comment:
`app` should be pointer, since that's what is returned by `cfenv.Current()`

Reply:
If it's a pointer it can be nil so by making it not a pointer I force the caller to pass it. However, I can change it to check for `nil`ness and just return a passthrough lookup. Fair enough I guess.